### DISCUSSION TITLE
Cannot read property 'key' of undefined - On LDAP Login / Windows AD

### DISCUSSION THREAD

**Discussion by Thraxll** [requarks/wiki#3484] [Help / Questions]
I have successfully configured LDAP authentication and it's working for the first few people I've tested. One of them got the error when logging in - "Cannot read property 'key' of undefined". What should I check?
![2020-11-11 09_44_53-RE_ Wiki - Message (HTML)](https://user-images.githubusercontent.com/65030270/98832591-9c8db800-2402-11eb-8c89-2f4e16f13703.png)

Thanks!

**Comment by ira-fox-by** [comment]
I just spun up wikijs (v2.5.170) and seeing the same issue with LDAP/AD..  sometimes it works sometimes it throws the "Cannot read property 'key' of undefined..   I find if I go back and reselect the  AUTH provider at login it works...

**Comment by Thraxll** [comment]
> I just spun up wikijs (v2.5.170) and seeing the same issue with LDAP/AD.. sometimes it works sometimes it throws the "Cannot read property 'key' of undefined.. I find if I go back and reselect the AUTH provider at login it works...

This works. I've turned off local auth in favor of LDAP so I hope tis gets fixed soon!
